app/dashboard/helpers.py

```diff
@@ -452,13 +452,12 @@ def create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id):
 
         try:
             new_bounty = Bounty.objects.create(**bounty_kwargs)
+            featured_funded_bounty('founders@gitcoin.co', new_bounty)
```

Comment:
This is still going to trigger every time the bounty changes - the reason for this is that this function is called every time a bounty changes, not just when the bounty is created for the first time (the function name `create_new_bounty` is not a great description of what this function does). I think you'd be better off putting this call in a post-save signal on the `Bounty` model: https://docs.djangoproject.com/en/2.1/ref/signals/#post-save <-- the `created` argument will tell you whether or not this bounty was created for the first time.

Reply:
@iamonuwa lets get this resolved and then I can payout! Thanks!

Reply:
> This is still going to trigger every time the bounty changes - the reason for this is that this function is called every time a bounty changes, not just when the bounty is created for the first time (the function name `create_new_bounty` is not a great description of what this function does). I think you'd be better off putting this call in a post-save signal on the `Bounty` model: https://docs.djangoproject.com/en/2.1/ref/signals/#post-save <-- the `created` argument will tell you whether or not this bounty was created for the first time.

Fixing this now